.github/workflows/updatejson.yml

```diff
@@ -0,0 +1,27 @@
+name: Housekeeping
+on: 
+  push:
```

Comment:
Note: I have scheduled this pipeline to run when A) someone pushes directly to master and B) when its the 1st and 15th of the month. I have A in because Github scheduled actions can be quite flaky in starting (ie lots of posts like https://stackoverflow.com/questions/59560214/github-action-works-on-push-but-not-scheduled). Therefore it can be useful to have a manual way of running this action to help check/kick off the scheduled job. 

B is a arbitrary timeline picked by me. There are restrictions on how long github actions can take, but this script takes a few minutes to run and therefore shouldn't hit any restrictions. If need be, I can change these scheduled times easily.